x/ethbridge/types/msgs.go

```diff
@@ -15,11 +15,25 @@ import (
 type MessageType uint32
 
 const (
-	MsgSubmit     MessageType = 0
-	MsgRevert     MessageType = 1
-	MsgReturnCeth MessageType = 2
+	MsgSubmit          MessageType = 0
+	MsgRevert          MessageType = 1
+	MsgReturnCeth      MessageType = 2
+	InvalidMessageType MessageType = 3
 )
 
+// MessageTypeStringToEnum return MessageType according to input string
+func MessageTypeStringToEnum(messageType string) MessageType {
+	switch messageType {
+	case "submit":
+		return MsgSubmit
+	case "revert":
+		return MsgSubmit
+	case "return":
+		return MsgSubmit
+	}
```

Comment:
In call cases except the case that no cases are met, the same type is returned. I do not understand this.

Reply:
It is a bug, will fix it.